ci-operator/step-registry/optional-operators/subscribe/optional-operators-subscribe-commands.sh

```diff
@@ -129,7 +132,13 @@ EOF
 
 echo "CatalogSource name is \"$CATSRC\""
 
-IS_CATSRC_CREATED=false
+else
```

Comment:
Can you add a comment describing what we're trying to do here

Reply:
@nmars 
Whenever, the script is ran for the first time on a cluster 
a catalog source is being created. 
Within CVP since it always creates a fresh cluster its not a problem. 
however, if we would like to run the above script again and again in different namespace.
It would error out, since catalog source has already been created. 
We are bypassing it by a if condition based on IS_CATSRC_CREATED which is defaulted to false. 
It will be true while running it locally.

Reply:
just out of curiosity, what the use case for running the script again and again n the same cluster?

Reply:
It for development purposes, If we would like to run the scripts locally we do not want a new cluster everytime.

Reply:
@apodhrad